R/layers.R

```diff
@@ -1128,6 +1128,7 @@ addPolylines <- function(
   fill = FALSE,
   fillColor = color,
   fillOpacity = 0.2,
+  offset = NULL,
```

Comment:
I think new parameters should be added at the end to keep backward compatibility for positional arguments.

Reply:
The documentation for `offset` is missing.

Reply:
I've added the documentation for the parameter but I'm not sure what's the best way to update the `man` files. Is it just a case of using `devtools::document()` ?

Reply:
yes, or you can change the settings in RStudio under Build > More > Configure Build Tools. In the Panel under "Build Tools" you can click the "Configure" Button and check "Install and Restart".

Reply:
If you want to leave the dataset "corunaroads" in the PR, then you need to document that aswell to pass the R checks. You can see the documentation for the other datasets in `/R/data.R`. Then document & reinstall and resubmit the changes.

Then you should pass the Travis Checks for R. I'm not sure about the Javascript Check.